test/toTex_test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const parse = require('../docs/bundle.js');
+const toTex = require('../lib/toTex.js');
```

Comment:
You'll need to `npm run build` or `npm run watch`?  It should update `bundle.js`.  I usually run `npm run watch` as I work so that it continually updates `bundle.js`.